packages/clay-card/src/Card.tsx

```diff
@@ -56,7 +57,7 @@ const ClayCard: React.FunctionComponent<IProps> & {
 		user: displayType === 'user',
 	};
 
-	const TagHeaderName = interactive ? 'a' : 'div';
+	const TagHeaderName = interactive ? (onClick ? 'span' : 'a') : 'div';
```

Comment:
I think it's more semantic to render a `button` when it has an `onClick`.

Reply:
oh true, not sure why i didn't think of that.